packages/@aws-cdk/aws-servicecatalogappregistry/test/integ.attribute-group.ts

```diff
@@ -20,5 +21,11 @@ new appreg.AttributeGroup(stack, 'TestAttributeGroup', {
     },
   },
 });
+const myRole = new iam.Role(stack, 'MyRole', {
+  assumedBy: new iam.AccountPrincipal(stack.account),
+});
+attributeGroup.shareResource({
+  roles: [myRole],
+});
```

Comment:
Could we get a share with multiple roles?

Reply:
Added this in to the integ test.